tests/systemxhr.js

```diff
@@ -0,0 +1,50 @@
+// mozSettings API tests (polyfilled!)
```

Comment:
Please let the tests only on the gh-pages branch! (remove it from here, and do a PR to gh-pages with the test only)